### PR TITLE
Make email signatures bit more general

### PR DESCRIPTION
**Pull request by dz0ny**

### CODE REVIEW

File: onboarding.md [hunk 1]
```diff
@@ -228,36 +228,47 @@ Regards
 A few examples to base your signature off of. Choose your own title, it can be whatever you wish.
 
 ```
-Dejan Murko
+Gavin Belson
```
Comment (dz0ny) [inline onboarding.md]: Or maybe Erlich Bachman 🚬

**Comment by dmurko** [comment]
@dz0ny Reason? I usually prefer personal over general for these things.

**Comment by dz0ny** [comment]
I prefer more general ones since they are templates.

**Comment by dz0ny** [comment]
Personal I don't want to be listed here :)

**Comment by dmurko** [comment]
OK, I'll remove you and keep me and zupo.

**Comment by zupo** [comment]
I kind of agree with @dz0ny, I would prefer my email not to be published so plainly.  Something like this?

```
Jane Doe

Lead Awesomeness Officer
---------------
Niteo

P: +48 512 0123
E: jd@teamniteo.com
W: www.teamniteo.com
```

**Comment by dmurko** [comment]
Oh well, as you wish. I'm excluding myself from this.

**Comment by dz0ny** [comment]
And mind the fact this will be public...

**Comment by dmurko** [comment]
It already is.

**Comment by karantan** [comment]
Yeah, definitely emails shouldn't be public. We will get spammed to infinity

**Comment by dmurko** [comment]
This was done in different PR.

**Pull request closed without merging**